### DISCUSSION TITLE
Laravel Passport with Laravel 10 Route

### DISCUSSION THREAD

**Discussion by kingrootid** [laravel/framework#47513] [Q&A]
Hello

i want to ask about registering route laravel passport with laravel 10 

i write on 
App/Providers/AuthServiceProvide.php 
on function Boot
`Passport::routes();`

and i got the error 

  Call to undefined method Laravel\Passport\Passport::routes()

  at app/Providers/AuthServiceProvider.php:31
     27▕      */
     28▕     public function boot(): void
     29▕     {
     30▕         $this->registerPolicies();
  ➜  31▕         Passport::routes();
     32▕ 
     33▕         Passport::personalAccessTokensExpireIn(now()->addDays(1));
     34▕     }
     35▕ }

      +7 vendor frames 

  8   [internal]:0
      Illuminate\Foundation\Application::Illuminate\Foundation\{closure}(Object(App\Providers\AuthServiceProvider))
      +5 vendor frames 

  14  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))

**Comment by schonhoff** [comment]
Hello,

do you use Passport v11? Maybe look here:
https://github.com/laravel/passport/pull/1464
They changed it on v11 that you do not need to put this in anymore.

**Comment by KateKoltsova** [comment]
As far as I know, this function no longer needs to be registered, when installing the library, routes are generated automatically, you can check this in Artisan
